api/client/samples/crop_models/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3623 @@
+{
```

Comment:
<p>This step failed for me (windows python 3) with this error:</p><p> </p><p><strong>NameError</strong>                                 Traceback (most recent call last)</p><p><strong>&lt;ipython-input-39-94a35a406809&gt;</strong> in &lt;module&gt;</p><p>      2 ranges <strong>=</strong> <strong>[(100,</strong> <strong>190),</strong> <strong>(190,</strong> <strong>265),</strong> <strong>(265,350)]</strong></p><p>      3 <strong>for</strong> l<strong>,</strong> r <strong>in</strong> ranges<strong>:</strong></p><p><strong>----&gt; 4     </strong>sub_dat <strong>=</strong> df_plot<strong>[</strong>df_plot<strong>['DOY']</strong> <strong>&gt;</strong> l<strong>][</strong>df_plot<strong>['DOY']</strong> <strong>&lt;</strong> r<strong>]</strong></p><p>      5     sub_dat<strong>['intercept']</strong> <strong>=</strong> <strong>1.</strong></p><p>      6     X <strong>=</strong> sub_dat<strong>[['lst','pet','trmm','ndvi','intercept']]</strong></p><p> </p><p><strong>NameError</strong>: name 'df_plot' is not defined</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>Removed this excess cell</p>